### ISSUE TITLE
Validation errors when using true or false in the column name

### ISSUE THREAD

**Issue by abdullahahmeda** [zenstackhq/zenstack#522]
**Description and expected behavior**
I have the following schema for true & false questions:
```prisma
model Question {
  id           Int            @id @default(autoincrement())
  text         String         @db.Text()
  trueText     String?        @db.Text()
  falseText    String?        @db.Text()
  answer       String         @db.Text()
}
```
When I run `npx zenstack generate` I get the following error:
```
Validation errors:
line 79: Expecting token of type '}' but found `true`. [true]
line 80: Expecting token of type '}' but found `false`. [false]
```
Also the zenstack vs code extension is linting the error:
![image](https://github.com/zenstackhq/zenstack/assets/23556281/163f9682-1760-4d8f-9f35-be6636f5ca7c)

**Environment:**

-   ZenStack version: 1.0.0-beta.2
-   Prisma version: 4.9.0
-   Database type: Postgresql

**Comment by jiashengguo** [comment]
@abdullahahmeda   It's a bug for sure, thanks for reporting it!  We will make sure it is fixed in the next release.

**Comment by abdullahahmeda** [comment]
Thank you.
For now I fixed this error by renaming `falseText` to `textForFalse` and `trueText` to `textForTrue`.

**Comment by ymc9** [comment]
Fixed by #530
